src/main/java/the_warlord/cards/warlord/TripleSlashCopy.java

```diff
@@ -0,0 +1,45 @@
+package the_warlord.cards.warlord;
+
+import com.megacrit.cardcrawl.actions.AbstractGameAction;
+import com.megacrit.cardcrawl.actions.common.DamageAction;
+import com.megacrit.cardcrawl.cards.DamageInfo;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import the_warlord.WarlordMod;
+import the_warlord.cards.CustomWarlordModCard;
+import the_warlord.characters.Warlord;
+
+public class TripleSlashCopy extends CustomWarlordModCard {
```

Comment:
This class can be deleted if you use the CardMetaUtils::playCardAdditionalTime approach.